dune_daq_services/runservices/postgresql/postgresql/helm-values.yml

```diff
@@ -11,9 +11,6 @@ serviceBindings:
   enabled: true
 
 primary:
-  initdb:
-    args: '--data-checksums --locale=en_US.UTF-8 --set huge_pages=off'
```

Comment:
I removed these because for some reason it was usingan old runnumber sql table as a configMap and wouldn't let me run runservices-postgresql-0 without that mapping https://github.com/DUNE-DAQ/daq-kube/tree/503bc12977727423c6f159a019c5e7384edfa4a7/dune_daq_services/runservices/postgresql/config

Reply:
Is it possible to keep this without this

scriptsConfigMap: runnum-table-sql

Reply:
I'm thinking pocket may be reusing the local datafiles on your system.